iceoryx_platform/win/include/iceoryx_platform/unistd.hpp

```diff
@@ -18,6 +18,7 @@
 #define IOX_HOOFS_WIN_PLATFORM_UNISTD_HPP
 
 #include "iceoryx_platform/types.hpp"
+#include "iceoryx_platform/unistd.hpp"
```

Comment:
Please remove this. The header would include itself. The issue is that the include was removed from `unistd.cpp`